Examples/GenericSerialization.cs

```diff
@@ -47,7 +47,7 @@ void Start()
             //Log.Debug("GenericSerialization", "main skill: {0}", uDefinedObject["name"]);
             //Log.Debug("GenericSerialization", "test: {0}", messageResponse);
```

Comment:
ooh man, i think we can delete all of this commented code. sorry, lol!

Reply:
deleted!